tests/behaviour/inspectcli.feature

```diff
@@ -1,5 +1,6 @@
 Feature: Behaviour testing of csvcubed inspect.
 
+    @wip
```

Comment:
Can you remove this? It'll make other devs lives harder if there are multiple `@wip` tags hidden about.